### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by cnasikas**

### CODE REVIEW

**Comment by dionyziz** [comment]
Please include a screenshot for each visual change in visual code reviews.

**Comment by cnasikas** [comment]
screen shot 2018-05-21 at 17 10 06

File: app/src/assets/css/App.css [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
-  font-family: sans-serif;
+  font-family: 'Roboto', sans-serif;
+}
+
+.wrapper {
+  flex-grow: 1;
+  z-index: 1;
+  overflow: 'hidden';
+  position: 'relative';
+  display: 'flex';
```
Comment (dionyziz) [inline app/src/assets/css/App.css]: no need for quotes in these

Comment (cnasikas) [reply app/src/assets/css/App.css]: done

Comment (cnasikas) [reply app/src/assets/css/App.css]: Btw I add stylelint and didn't complain

Comment (dionyziz) [reply app/src/assets/css/App.css]: It's not wrong, it's just not needed.

**Pull request merged**